apps/generic_terminal/generic_unix_shell.py

```diff
@@ -1,11 +1,39 @@
+from pathlib import Path
+
 from talon import Context, Module, actions
 
+from ...code.user_settings import get_list_from_csv
+
 ctx = Context()
 mod = Module()
 ctx.matches = r"""
 tag: user.generic_unix_shell
 """
 
+path = Path(__file__).parents[0]
+with open(path / "default_unix_utilities.csv") as f:
+    # get_list_from_csv() expects the columns in the reverse order
+    default_unix_utilities = {
+        spoken.strip(): command.strip()
+        for command, spoken in [line.split(",", 1) for line in f]
+    }
```

Comment:
This might be simpler if we use Python's `csv` module:

```
    with open(...) as f:
        default_unix_utilities = list(csv.reader(f))
```

Reply:
Good point. Used this in the current revision.